torch_xla/csrc/runtime/xla_coordinator.h

```diff
@@ -12,11 +13,17 @@ namespace runtime {
 // XlaCoordinator serves as the point of entry for all operations which
 // required the XLA distributed runtime, such as preemption coordination.
 class XlaCoordinator {
+ private:
+  // Private struct for making the constructor private, but still callable
+  // with std::make_unique<T>() function.
+  struct PrivateUse {
+    explicit PrivateUse() = default;
```

Comment:
Nit: this line can be removed.

Reply:
Do you mean the declaration of the explicit constructor?

If so, the problem is that we will allow `XlaCoordinator({})` constructor to be called. Making the default constructor explicit means that this class can only be instantiated by `XlaCoordinator::Create(...)`.